ci/release_notes.md

```diff
@@ -0,0 +1,3 @@
+# Improvements
+
+- Add new session_timeout params
```

Comment:
Be a little more explicit here.  It's _kind of_ duplicating information from the README, but the changelog will stay frozen in time on the releases page, whereas the README may change in the future.  Talk about what the param does and why.